pages/common/unflatten.md

```diff
@@ -0,0 +1,17 @@
+# unflatten
+
+> Adjust directed graphs to improve the layout aspect ratio.
+> Graphviz filters: `acyclic`, `bcomps`, `comps`, `edgepaint`, `gvcolor`, `gvpack`, `mingle`, `nop`, `sccmap`, `tred`, & `unflatten`.
+> More information: <https://www.graphviz.org/pdf/unflatten.1.pdf>.
+
+- Adjust directed graphs to improve the layout aspect ratio:
+
+`unflatten {{path/to/input1.gv}} {{path/to/input2.gv}} > {{path/to/output.gv}}`
+
+- Use `unflatten` as a preprocessor for `dot` layout:
```

Comment:
This explaines how the command works, but not really what it does.

Reply:
There isn't enough room to explain that it adds invisible edges (lines) and adjusts minimum lengths to make a long, thin layout more square. I added "to improve aspect ratio".